popolo/importers/popit.py

```diff
@@ -100,6 +100,12 @@ def update_optional_area(object_data):
                 #if we have an area_id instead of an inline area
                 area = self.get_existing_django_object('area', area_id)
             return area
+        self.events = {}
+        # store events in a temp local dict:
+        if 'events' in data:
+            event_id_to_django_object = {}
```

Comment:
I don't think you need this `event_id_to_django_object` variable with the code as it is at the moment - it's never used.